scripts/util/conversionDNATable.js

```diff
@@ -0,0 +1,7 @@
+define({
```

Comment:
Thats to be discuss. How do we handle "genes" for the "dna". Are we hardcoding a list or are we finding a way to handle emerging genes.